.vscode/tasks.json

```diff
@@ -19,7 +19,9 @@
       "label": "Pytest",
       "type": "shell",
       "command": "pytest --timeout=10 tests",
-      "dependsOn": ["Install all Test Requirements"],
+      "dependsOn": [
```

Comment:
Please don't change this file.

Reply:
So I  didn't, I'll I did was open it and then the hooks formatted the file. 

I manually went back in and reset the changes on the file, and the hooks still reformatted it.

Reply:
Try reverting the changes and commit that without the commit hooks.